plugins/node/opentelemetry-instrumentation-express/.tav.yml

```diff
@@ -1,5 +1,5 @@
 express:
   - versions:
-      include: "^4.16.2"
+      include: ">=4.16.2 <6"
```

Comment:
`4.16.2` is not supported anymore, ideally bump this to `4.21.2`.

Reply:
True, although if 4.16.2 still works without issue I'm okay keeping this until it causes us problems, unless it is massively increasing testing time.

Reply:
I just mentioned it because of these: https://github.com/expressjs/express/security/advisories?state=published

Reply:
I guess I should have posted more context. I doubt any impact from CVEs here. I just like to point it out in case there are parts of the systems I am unaware of that if it is not done for a *real* reason it is best to just avoid the versions with published CVEs. Hope that helps clarify my reasoning for mentioning it.

Reply:
> downloads tab on npm

Ah yep, I meant specifically folks who use this instrumentation. But to your point, this version is overall pretty low in terms of downloads compared to even the next minor release. Besides, if someone is still on this old of a version of express, they are less likely to be upgrading this dependency. Let's increase minimum testing version 👍